frontend/config/stakingPrograms/base.ts

```diff
@@ -12,12 +12,12 @@ import { StakingProgramMap } from '.';
 
 export const BASE_STAKING_PROGRAMS_CONTRACT_ADDRESSES: Record<string, Address> =
   {
-    [StakingProgramId.MemeBaseAlpha]:
-      '0x06702a05312091013fdb50c8b60b98ca30762931',
+    [StakingProgramId.MemeBaseAlpha2]:
```

Comment:
the name is from here https://gateway.autonolas.tech/ipfs/f01701220a4281ec71a254c8dd08d624ff09609dc3235a0e63b05c1e410eb7e1f3d07e57d